sentry-jmx/src/main/java/io/sentry/jmx/AbstractSentryConfiguration.java

```diff
@@ -0,0 +1,53 @@
+package io.sentry.jmx;
+
+import io.sentry.SentryClient;
+import javax.management.InstanceAlreadyExistsException;
+import javax.management.MBeanRegistrationException;
+import javax.management.MBeanServer;
+import javax.management.MalformedObjectNameException;
+import javax.management.NotCompliantMBeanException;
+import javax.management.ObjectName;
+import javax.management.StandardMBean;
+
+public abstract class AbstractSentryConfiguration implements SentryConfigurationMXBean {
```

Comment:
I'm just curious, is your main usecase changing the DSN? I'm trying to think of when the release/dist/servername would change for a running process. Of all of those I guess servername seems the most possible but also kind of weird.

Reply:
Actually yes. Another useful usecase would probably be the MDC/normal tags.

Reply:
The currently exposed settings are only meant as an example.
Before I have to write these for all the settings and later we then decide on doing it via reflection or not at all.